_blog/2018/aws-nlb.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Configure Istio Ingress with AWS NLB"
+overview: Describes how to configure Istio ingress with an external load balancer Network Load Balancer on AWS
+publish_date: April 10, 2018
+subtitle: Ingress AWS Network Load Balancer
+attribution: Julien SENON
+
+order: 90
+
+layout: blog
+type: markdown
+redirect_from: "/blog/aws-nlb.html"
+---
+{% include home.html %}
+
+Network load balancer (NLB) could be used instead of classical load balancer. You can find comparison between different AWS `loadbalancer` [here](https://aws.amazon.com/elasticloadbalancing/details/#compare).
```

Comment:
Please avoid links that say just "here". See https://istio.io/about/contribute/style-guide.html#create-useful-links for context